test/unit/app/models/landing_page/featured_block_test.rb

```diff
@@ -69,13 +68,19 @@ class FeaturedBlockTest < ActiveSupport::TestCase
   end
 
   test "valid when missing featured content blocks" do
-    subject = LandingPage::FeaturedBlock.new(@valid_featured_block_config, @valid_featured_images, nil)
+    subject = LandingPage::FeaturedBlock.new(
+      @valid_featured_block_config.except("featured_content"),
+      @valid_featured_images,
+    )
     assert subject.valid?
   end
 
   test "invalid when featured content blocks are invalid" do
     invalid_blocks_config = [{ "invalid" => "because I do not have a type" }]
-    subject = LandingPage::FeaturedBlock.new(@valid_featured_block_config, @valid_featured_images, invalid_blocks_config)
+    subject = LandingPage::FeaturedBlock.new(
+      @valid_featured_block_config.tap { _1["featured_content"]["blocks"] = invalid_blocks_config },
```

Comment:
I find this `_1` really unpleasant, excited for being able to use `it` instead in Ruby 3.4 🎉

Reply:
I've come to quite like it, mostly through trying to golf my way through advent of code. Agree `it` will be nicer.

I'd be happy to do this if it's less smelly:

```suggestion
      @valid_featured_block_config.tap { |config| config["featured_content"]["blocks"] = invalid_blocks_config },
```

Reply:
Or I could do this I think:

```suggestion
      @valid_featured_block_config.merge("featured_content" => { "blocks" => invalid_blocks_config }),
```